### DISCUSSION TITLE
Preview deployment for functions

### DISCUSSION THREAD

**Discussion by franciscohanna92** [vercel/community#531] [Help] [answered]
Hello!

I have a question for which I haven't found a clear answer anywhere.

When using the preview deployments for feature branches? Does this deploys preview versions for the functions or is it only the frontend part?

**Comment by swarnava** [comment, marked as answer]
Hi there,

Preview deployment also deploy preview version of Function during deployment

<img width="1058" alt="image" src="https://user-images.githubusercontent.com/1162991/161196826-1f4d44b4-f1ec-4d99-9728-af41ab54de7e.png">

Have a good one!
